src/pages/RepoPage/BundlesTab/BundleContent/AssetsTable/AssetsTable.spec.tsx

```diff
@@ -349,39 +352,39 @@ describe('AssetsTable', () => {
 
     describe('renders table rows', () => {
       it('renders asset column', async () => {
```

Comment:
curious why we have these tests split out into separate component renders vs. just confirming all the columns in a single test? This is a common pattern I've noticed throughout the codebase and just wanted some clarity on

Reply:
I personally prefer the more verbose style like the above, because it explains exactly what the test is checking, so if that test fails I know instantly what is going on. Whereas if it's checking all the columns I have to look deeper into which test failed.

As well it probably plays a bit nicer with test analytics so we can easily see which test is failing since we display it by test name/description.

Reply:
I see what you mean, though the error message from the test should still be quite clear when we do the expect() statement and a certain line fails.

The pro with this approach, as you mentioned, is if multiple columns are failing we see all fails at once. I'm fine either way, though I personally prefer consolidation for the reduced render and verbosity pros

Reply:
It does come at the cost of more tests/longer CI runs. Can't imagine we'd save a huge amount of time consolidating these though. If it were significant then it may not be worth the time saved debugging. Could be interesting to compare the runtimes.

Reply:
Also with the upcoming move to Vitest, we should hopefully see test CI times go down.

Reply:
Yeah at the end of the day it's mostly personal preference.

Knowing that our GH bill for the runners isn't too large either, that's a valid approach as well (in the short/med term at least)